updates to different articles chapter (#2409)
alan-turing-institute/the-turing-way: book/website/communication/dif-articles/reg.md

(cm-dif-articles-registered-reports)=
# Registered Reports

(cm-dif-articles-registered-reports-what)=
## What are Registered Reports?
Registered Reports are an article type divided into two published parts- a study protocol and a research article. 
The study protocol is reviewed **prior** to the data being collected and the research being carried out. 

> ```{figure} ../figures/cos-reg-reports.png

Reviewer: ```suggestion
```{figure} ../../figures/cos-reg-reports.png
```

---

> height: 500px

Reviewer: ```suggestion
height: 300px
```

> height: 300px

Reviewer: ```suggestion
height: 200px
```

This frees up researchers to be rewarded for simply doing good and transparent science without having to sell an impactful story about their research. 

> Registered reports are also a useful format for researchers who want to carry out replication studies and other novel, resource-intensive projects that may otherwise be too risky to attempt where successful publication is contingent on the results. More info can be found [here](https://royalsociety.org/blog/2016/11/registered-reports-what-are-they-and-why-are-they-important/).

Reviewer: ```suggestion
Registered reports are also a useful format for researchers who want to carry out replication studies and other novel, resource-intensive projects that may otherwise be too risky to attempt where successful publication is contingent on the results. More info can be found [in this Royal Society blog on registered reports](https://royalsociety.org/blog/2016/11/registered-reports-what-are-they-and-why-are-they-important/).
```
